cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,59 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('/')
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+      .click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`You clicked a button`)
+    });
   });
 
   it('should have the ability to assert scheduled allert', () => {
-
+    let alertDisplayed = false;
+    cy.get('#timerAlertButton')
+      .click();
+    cy.on('window:alert', (str) => {
+      alertDisplayed = true;
+      expect(str).to.equal(`This alert appeared after 5 seconds`)
+    });
   });
 
   it('should autimatically resolve alerts', () => {
-
+    cy.get('#confirmButton')
+      .click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`Do you confirm action?`)
+    cy.get('#confirmResult')
+      .should('contain', 'You selected Ok')
+    });
   });
 
   it('should have the ability to Cancel alerts', () => {
-
+    cy.get('#confirmButton')
+      .click();
+      cy.on('window:alert', (str) => {
+        expect(str).to.equal(`Do you confirm action?`)
+        expect(str).to.eq('second confirm')
+      cy.get('#confirmResult')
+        .should('contain', 'You selected Cancel')
+    });
   });
 
   it('should have the ability to enter text to alert', () => {
-
+    cy.get('#promtButton')
+      .click();
+    cy.on('window:prompt', (str) => {
```

Comment:
The event 'window:prompt' is stubbed but not asserted. Consider asserting the expected prompt text.